### ISSUE TITLE
Ties for longest path not being accounted for

### ISSUE THREAD

**Issue by Jacobtk** [bmcgary/TicketToRide#28]
According to the rules, if there exists a tie for longest path at the end of the game, each player who tied for longest path gets the 10 points.

**Comment by 198xiaozhu** [comment]
I haven't tested this one yet.

**Comment by 198xiaozhu** [comment]
quote from the internet "My wife and I bought Ticket to ride today. I’ve played many times before but it was my wife’s first time. She really really enjoyed it. On our second game, right before we started finalizing the scores, my marker was on 73 and she was on 74. We counted up our longest tracks to see who would win the longest train bonus. It was a tie, so we both won 10 points as it says in the rules. Then we added on the points we received for completing our route cards. It was a tie! We went back to the rules to see what we should to for a tie breaker. It says that in a tie whoever completed the most destination tickets wins. We both completed 3 destination tickets! The rules then state that “In the unlikely event that they are still tied, the player with the longest continuous path card wins.” Since we both had the longest continuous path, I think we broke the game… Anyone know who the winner is?"
